src/styles/designSystem.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from "classnames";
+
+interface HeaderProps {
+  toggleable?: boolean;
+  // add other properties as needed
+}
+
+// Customize PrimeReact components
+const designSystem = {
+  panel: {
+    header: ({ props }: { props: HeaderProps }) => ({
+      className: classNames(
```

Comment:
Wow this is actually pretty nice, way better than one long string with template literals. TIL